woocommerce/woocommerce
# Remove some usages of set-output

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .






### How to test the changes in this Pull Request:



Test the code freeze workflow:

1) Fork this repository.
2) Enable issues in your fork from `https://github.com/YOURORG/woocommerce/settings`
3) Enable actions in your fork from `https://github.com/YOURORG/woocommerce/settings/actions`. Also, make sure workflows have read and write permissions and the ability to create and approve PRs.
4) Enable the scheduled workflow from `https://github.com/YOURORG/woocommerce/actions/workflows/release-code-freeze.yml`.
6) Merge this PR into `trunk` of your new fork.
9) Run the action manually from `https://github.com/YOURORG/woocommerce/actions/workflows/release-code-freeze.yml` with "2022-07-14" as the value for time override. This should cause the code-freeze to run the next steps. 
10) Run the action manually from `https://github.com/YOURORG/woocommerce/actions/workflows/release-code-freeze.yml` with "2022-07-12" as the value for time override. Because this is not a code freeze date the next steps of the workflow should not run.

Test the monorepo setup:

1) CI checks passing as usual should be good enough to prove this is still working.

Test "prepare package release"

1) After making sure you set up actions in your fork, you can run the "Prepare package release" command with -a and with a package name in your fork, the action should perform correctly creating only a PR for the specific package you mentioned in override or for all of them if you left it as -a

Test "release changelog"

I think the changes are small enough here to warrant not testing the single migration of set-output. Otherwise you can test by running this workflow manually against a release in your fork.




### Other information:




### FOR PR REVIEWER ONLY:
